submission/submission-webapp/src/main/resources/application-local.yml

```diff
@@ -18,10 +18,10 @@ spring:
       max-request-size: 10240MB
 
 app:
-  basepath: # Path to the folder to be used as storage
-  filesDirPath: # Path to the folder to be used for the user/groups files
-  tmpDirPath: # Absolute of the temp folder to store files
+  basepath: # Absolute path to the folder to be used as storage
```

Comment:
clearly not part of the scope of this ticket but this weekend I learn that we can document property class and IDEa will provide nice autocompletion for properties so if you have time we can do it there.

Reply:
I think this file is more intended to run the application than to be used in the IDE. This is a good suggestion tho.